backend/services/services.py

```
@@ -8,7 +8,7 @@
 import secrets
 from pathlib import Path
 from datetime import datetime, timezone
-from api.v1.endpoints.auth import get_current_user
+from api.v1.endpoints.Auth.auth import get_current_user
```

Ruff (0.8.2)

`api.v1.endpoints.Auth.auth.get_current_user` imported but unused

Remove unused import: `api.v1.endpoints.Auth.auth.get_current_user`

(F401)